src/main/kotlin/bsu/cc/parser/XlsxParser.kt

```diff
@@ -149,3 +146,31 @@ private fun rowToCellMap(row: Row): Map<Int, Cell> {
 
     return output
 }
+
+fun highlightRow(
+        workbook: Workbook,
+        sheetIndex: Int,
```

Comment:
Any reason for switching from passing a sheet object to using the index?

Reply:
Just the redundancy of passing in a (potentially large) property of the worksheet with the object itself, as opposed to specifying how to get it from the worksheet.

Reply:
Does getSheetAt not return a reference to an internal object within the workbook?